spec/Section 6 -- Execution.md

```diff
@@ -143,6 +143,84 @@ ExecuteMutation(mutation, schema, variableValues, initialValue):
     selection set.
   * Return an unordered map containing {data} and {errors}.
 
+Unlike queries and mutations, subscriptions have a lifetime that consists of three
+phases. Between the the subscribe and unsubscribe phases, the subscription is
+considered to be in the "active" state.
+
+### Subscribe
+
+The result of executing a subscription operation is a subscription object with
+the following capabilities:
+
+  * **must** support observation of the associated publish stream (for example,
+  via iteration, callbacks, or reactive semantics).
+  * **must** support cancellation of the subscription (aka unsubscribe).
+  * **must** support a way to identify the subscriber/subscription pair (for
+    example, a GUID/callback table or closure over the callback).
+  * **may** include an initial response associated with executing the selection
```

Comment:
Capitalize "m" in "must"/"may" here for consistency with the other lists (that all seem to start with capitals)?